tests/resources/serializers/test_csl_serializer.py

```diff
@@ -39,3 +40,23 @@ def test_csl_serializer(
     serialized_record = serializer.dump_one(full_record)
 
     assert serialized_record == expected_data
+
+
+def test_csl_serializer(
+        running_app, full_record, vocabulary_clear):
+    """Test serializer to DayaCide 4.3 JSON"""
+    # if the record is created this field will be present
+    full_record["id"] = "12345-abcde"
+
+    expected_data = {'abstract': 'Test', 'language': 'dan',
+                     'issued': {'date-parts': [['2018'], ['2020', '09']]},
+                     'type': 'graphic',
+                     'note': 'Funding by European Commission ROR 1234.',
+                     'version': 'v1.0', 'title': 'InvenioRDM',
+                     'author': [{'family': 'Nielsen, Lars Holm'}],
+                     'id': '12345-abcde', 'publisher': 'InvenioRDM',
+                     'DOI': '10.5281/inveniordm.1234'}
+
+    serializer = StringCitationSerializer()
+    serialized_record = serializer.dump_one(full_record)
```

Comment:
This test is nice, I wonder if we should add a random one for a specific format... Although we are relying on an 3rd party dependency so we should not have to test it... WDYT?

Reply:
Yes and please add tests for the view_args param. Test that your code will not fail when:
* passing bad values for args `style` and `locale` (it should throw a 400 error)
* not passing at all `style` and `locale` (it should use default)